src/wolframite/wolfram.clj

```diff
@@ -6351,4 +6350,16 @@
   ;; If there is any w/ symbol such as w/Plus, replace it just with the fn name, `Plus` -
   ;; we need the whole body to be pure symbols and primitives only, which Wolfram will understand
   (let [symbolic-sexp (clojure.walk/prewalk wolframite.impl.wolfram-syms.intern/try->wolf-sym sexp)]
-    `(list (quote ~'Function) (quote ~args) (quote ~symbolic-sexp))))
\ No newline at end of file
+    `(list (quote ~'Function) (quote ~args) (quote ~symbolic-sexp))))
+(def >_< "Maps to the Wolfram function Simplify" (wolframite.impl.wolfram-syms.intern/wolfram-fn (quote >_<)))
```

Comment:
Where does this come from? Was it added manually, or...?

Reply:
This was the leftover from my first attempt at changing the symbol aliases. I realised afterwards that I'd overloaded some symbols though and changed it. Either I didn't rewrite wolfram... or I didn't commit the change.

Reply:
Mmm there's something else going on as changes I'm sure I made are not here. I wonder if updated the wrong branch? Anyways, I'll sort it out now.